lib/tasks/send_survey.rake

```diff
@@ -0,0 +1,27 @@
+namespace :survey do
+  desc "send survey"
+  task :send => :environment do
+    # invoke script with rake survey:send EXCLUDEFILE=/path/to/file.txt
+    # read list of users to exclude
+    # should contain one user-ID per line
+    if ENV['EXCLUDEFILE']
+      exclude_users = File.readlines(ENV['EXCLUDEFILE']).each {|l| l.chomp!}
```

Comment:
You might want to use `map` instead of `each`. `each` just iterates over the elements of the Array and returns `nil`.